package.json

```diff
@@ -84,7 +84,7 @@
   "author": "Juan Cruz Viotti <juan@balena.io>",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.20.0 <15.0.0"
+    "node": ">=10.20.0"
```

Comment:
As this PR is now for `os configure` and `local flash` only, I think this commit (that updates `package.json` `node` versions, and `INSTALL-ADVANCED.md`) should be dropped from this PR and added to the other PR that resolves `balena push` with Node.js v14.
This is because, after this PR is merged, I understand we will still have to ask users to use Node.js v10 or v12 only (like the balena team, using `balena push` with the devenv / BoB).

Reply:
Btw, my commit re `net-keepalive` would also fit better the PR for `balena push` than the PR for `os configure` and `local flash`, but it should also be harmless to have it here. It's up to you.

Reply:
I agree with the first point. I'd say keep the `net-keepalive` bump. Also, we could probably edit the issue about the segfault on push. Nothing segfaults now.

Reply:
> we could probably edit the issue about the segfault on push. Nothing segfaults now.

Issue #2200, right? Were you able to reproduce it? So that we could demonstrate that it crashed before this PR, and not after this PR. I wasn't able to reproduce it myself and I'm thinking I may have jumped to conclusions when I assumed that issue #2200 was related to Node.js v14.

Reply:
By the way, the first commit still claims `Resolves: #2079`, but I think it should not. There are users watching issue #2079, which I think was also linked in the forums and in homebrew, as the issue that makes the balena CLI fully support Node.js v14. I think we need to leave that issue open until `balena push` is resolved in another PR.